.github/workflows/test.yml

```diff
@@ -26,11 +26,24 @@ jobs:
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
-      
+
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Install unixodbc on AMD
+        if: matrix.architecture == 'amd64'
+        run: |
+          sudo apt install unixodbc
+
+      - name: Install unixodbc on ARM
+        if: matrix.architecture == 'arm64'
+        run: |
+          brew install unixodbc
+          unixodbc_version="$(odbcinst --version | grep -Eo '[0-9].*')"
+          export LDFLAGS="-L/opt/homebrew/Cellar/unixodbc/$unixodbc_version/lib"
+          export CPPFLAGS="-I/opt/homebrew/Cellar/unixodbc/$unixodbc_version/include"
```

Comment:
Why do we need these?

Reply:
@antepusic can you just confirm on this one that we don't need, why did you put that here?